sdcflows/interfaces/utils.py

```diff
@@ -267,6 +267,26 @@ class DenoiseImage(_DenoiseImageBase, _CopyHeaderInterface):
     _copy_header_map = {"output_image": "input_image"}
 
 
+class PadSlicesInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc="3D or 4D NIfTI image")
+
+
+class PadSlicesOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc="The output file with even number of slices")
+    padded = traits.Bool(desc="Indicator if the input image was padded")
+
+
+class PadSlices(SimpleInterface):
+    input_spec = PadSlicesInputSpec
+    output_spec = PadSlicesOutputSpec
```

Comment:
```suggestion
    input_spec = _PadSlicesInputSpec
    output_spec = _PadSlicesOutputSpec
```

In niworkflows, and here, we use the underscore so then sphinx renders the interface (using nipype's extension of sphinx) more nicely.